Source/WebCore/page/scrolling/ScrollingStateScrollingNode.h

```diff
@@ -45,6 +45,11 @@ struct ScrollbarHoverState {
     friend bool operator==(ScrollbarHoverState const& lhs, ScrollbarHoverState const& rhs) = default;
 };
 
+struct MouseLocationState {
```

Comment:
Should this be merged with `ScrollbarHoverState`?

Reply:
Since these states are not coupled I think it maks sense to keep them seperate. If we end up coupling them like you suggested above, then I think that would be reasonable.